### ISSUE TITLE
rowClick executes both onSelectRow and onRowClick

### ISSUE THREAD

**Issue by tbaeg** [AllenFang/react-bootstrap-table#128]
When I click the checkbox element it should only tigger the `onSelectRow` function. It executes both `onRowClick` and on `onSelectRow`. Found the problem.. silly issue!

This is wrong..
```
  rowClick(e){
    if(e.target.tagName !== "INPUT")
      if (this.props.selectRow && this.props.selectRow.clickToSelect) this.props.onSelectRow(e.currentTarget.rowIndex, !this.props.isSelected);
      if (this.props.onRowClick) this.props.onRowClick(e.currentTarget.rowIndex);
  }
```

Should be..
```
  rowClick(e){
    if(e.target.tagName !== "INPUT") {
      if (this.props.selectRow && this.props.selectRow.clickToSelect) this.props.onSelectRow(e.currentTarget.rowIndex, !this.props.isSelected);
      if (this.props.onRowClick) this.props.onRowClick(e.currentTarget.rowIndex);
    }
  }
```

Or else there is some very weird behavior.

Braces are important, ha.

**Comment by tbaeg** [comment]
I probably should've wrote some `jest` test for them. If I get some time, I'll do that.

**Comment by AllenFang** [comment]
oh god, I can't believe I make this shame mistake ;(

**Comment by tbaeg** [comment]
No problem! I made the mistake first, haha. Easy fix. :)

**Comment by AllenFang** [comment]
Thanks :) I've merged and fixed it on v1.2.8
